src/gc/proxy.d

```diff
@@ -111,18 +104,7 @@ extern (C)
 
     void gc_init()
     {
-        version (GCCLASS)
-        {   void* p;
-            ClassInfo ci = typeid(GC);
-
-            p = malloc(ci.init.length);
-            (cast(byte*)p)[0 .. ci.init.length] = ci.init[];
-            _gc = cast(GC)p;
-        }
-        else
-        {
-            _gc = cast(GC*) calloc(1, GC.sizeof);
-        }
+        _gc = cast(GC*) calloc(1, GC.sizeof);
```

Comment:
You should still copy the init value over the allocated data. There is at least `gcversion` with a non-zero initializer.